# HumanRejectedException â€” ðŸ¦œðŸ”— LangChain  documentation

**URL:** https://python.langchain.com/api_reference/community/callbacks/langchain_community.callbacks.human.HumanRejectedException.html
**Word Count:** 18
**Links Count:** 181
**Scraped:** 2025-07-21 08:02:42
**Status:** completed

---

# HumanRejectedException\#

_class _langchain\_community.callbacks.human.HumanRejectedException[\[source\]](https://python.langchain.com/api_reference/_modules/langchain_community/callbacks/human.html#HumanRejectedException)\#     

Exception to raise when a person manually review and rejects a value.

__On this page